OpenSim/Sandbox/MatlabScripts/timesSeriesTable.m

```diff
@@ -0,0 +1,195 @@
+function output_table = timesSeriesTable(generictable)
+%% // Convert between OpenSim and Matlab Timeseries 
+%  
+%   Inputs can be either an OpenSim TimeSeriesTable, TimesSeriesTableVec3,
+%   or a Matlab times series collection. 
+%
+%   Outputs from OpenSim TimesSeriesTable's will be a Matlab times series
+%   collection. 
+%
+%   Function does not maintain meta data between types. 
+%
+
+% Author: James Dunne, Shrinidhi K. Lakshmikanth, Chris Dembia, Tom Uchida,
+% Ajay Seth, Ayman Habib, Jen Hicks. 
+
+import org.opensim.modeling.*
+
+if strcmp( class(generictable), 'org.opensim.modeling.TimeSeriesTableVec3') ...
+   || strcmp( class(generictable), 'org.opensim.modeling.TimeSeriesTable') 
+   
+    output_table = table2tsc(generictable);
+
+elseif  strcmp( class(generictable), 'tscollection') 
+   
+    output_table = tsc2table(generictable);
+    
+else
+    error('input must either be an OpenSim TimeSeriesTable or a Matlab tscollection')
+end
+
+end
```

Comment:
Is there a particular reason for using 1 function to support both directions of conversions? I think I prefer having two separate functions. That way, user code is less ambiguous:

```matlab
table_out = timeSeriesTable(table_in)  % which one is the OpenSim DataTable?
```

What about `convertFromOpenSimTimeSeriesTable()` and `convertToOpenSimTimeSeriesTable()`?